tests/laser/evm_testsuite/evm_test.py

```diff
@@ -76,6 +76,8 @@ def test_vmtest(
     # Arrange
     if test_name == "gasprice":
         return
+    if "calldatacopy0_return" not in test_name:
```

Comment:
this will run only those tests which have `calldatacopy0_return` in them right?

Reply:
oops